packages/fetchai/connections/p2p_libp2p/aea/api.go

```diff
@@ -51,6 +51,15 @@ type Pipe interface {
 	Close() error
 }
 
+// Needed to break import cycle
+type AgentRecord struct {
+	ServiceId     string
```

Comment:
[LPR5]
serviceId is the message to be appended to public key for signing.
currently not used